lib/Mojo/IOLoop/Subprocess.pm

```diff
@@ -50,7 +51,7 @@ sub _start {
   unless ($pid) {
     eval {
       $self->ioloop->reset;
-      my $results = eval { [$self->$child] } || [];
+      my $results = eval { [$self->$child] } // Mojo::Util::_EMPTY_ARRAY;
```

Comment:
Isn't defined or slower than or?

Reply:
No, `//` only has to check the `SVf_OK` flag, while `||` has to check the actual value to see if it's truthy.